### PR TITLE
refactor: split some contents from quick-start

### PR DESCRIPTION
**Pull request by RetricSu**
- add blockchain-network page
- add cli tool page

### CODE REVIEW

File: website/docs/getting-started/blockchain-networks.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  - https://mainnet.ckb.dev/rpc
+- Guide: [Running CKB Mainnet Node](/docs/node/run-mainnet-node)
+- Mainnet Explorer: [https://explorer.nervos.org/](https://explorer.nervos.org/)
+- Address Prefix: `ckb`, eg `ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqxwquc4`
+
+### CKB Testnet: PUDGE
+
+In addition to Mainnet, there is a public testnet for CKB called PUDGE. The testnet is used by developers to test both protocol upgrades as well as potential <Tooltip>Scripts</Tooltip>(smart contract) in a production-like environment before deployment to Mainnet. Think of this as an analog to production versus staging servers.
+
+You should test any Script code you write on a testnet before deploying to Mainnet. Many dApps also runs on testnet before going to production level.
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:33]:
```suggestion
In addition to Mainnet, there is a public Testnet for CKB called PUDGE. The Testnet is used by developers to test both protocol upgrades as well as potential <Tooltip>Scripts</Tooltip>(smart contract) in a production-like environment before deployment to Mainnet. Think of this as analogous to production versus staging servers.

You should test any Script code you write on a Testnet before deploying to Mainnet. Many dApps also run on Testnet before going to the production level.
```

File: website/docs/getting-started/blockchain-networks.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+Most of the times, you will need some Testnet tokens to deploy Scripts & test your dApp.
+
+You can get free Testnet tokens from the faucet website:
+
+- [https://faucet-api.nervos.org/](https://faucet-api.nervos.org/)
+
+## Private Networks
+
+Besides public networks that anyone can join, you can also running your own priate CKB blockchain on your local machine called Devnet(developmenmt network). Most developers use the CKB Devnet as the local developmnent environment.
+
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:51]:
```suggestion
Besides public networks that anyone can join, you can also run your own priate CKB blockchain on your local machine, called Devnet (development network). Most developers use the CKB Devnet as the local development environment.

```

File: website/docs/getting-started/blockchain-networks.mdx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+CKB Devnet Address Prefix: `ckt`, eg `ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqxwquc4`
+
+## Switch Between Different Networks
+
+One of the things you might want to do is switching between different blockchain networks during the development of your dApps. The recommended order is:
+
+1. Firstly, develop your dApp in the CKB Devnet, where everything is under your control with a rapid feedback loop.
+2. Secondly, migrate your dApp to the CKB Testnet to see if it works fine in a more production-like environment.
+3. Lastly, launch your dApp on the CKB Mainnet for production.
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:67]:
```suggestion
1. Develop your dApp in the CKB Devnet, where everything is under your control with a rapid feedback loop.
2. Migrate your dApp to the CKB Testnet to see if it works fine in a more production-like environment.
3. Launch your dApp on the CKB Mainnet for production.
```

File: website/docs/getting-started/blockchain-networks.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+To switch different networks, you need to point your dApp to different blockchain RPC urls with different pre-deployed System Script configs.
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:69]:
```suggestion
1. Develop your dApp in the CKB Devnet, where everything is under your control with a rapid feedback loop.
2. Migrate your dApp to the CKB Testnet to see if it works fine in a more production-like environment.
3. Launch your dApp on the CKB Mainnet for production.

To switch different networks, you need to point your dApp to different blockchain RPC URLs with different pre-deployed System Script configs.
```

File: website/docs/getting-started/blockchain-networks.mdx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+```js title='Mainnet Example Config'
+const lumos = require("@ckb-lumos/lumos");
+const indexer = new lumos.Indexer("https://mainnet.ckb.dev/rpc");
+const rpc = new lumos.RPC("https://mainnet.ckb.dev/rpc");
+lumos.config.initializeConfig(lumos.config.Mainnet);
+```
+
+### Using OffCKB boilerplate
+
+OffCKB dApp boilerplates provide easy way to switch different networks by applying different environment variables in the project.
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:91]:
```suggestion
### Using OffCKB Boilerplate

OffCKB dApp boilerplates provide an easy way to switch different networks by applying different environment variables in the project.
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+
+# CLI Tools
+
+Commandline tools are often used by developers to perform specific blockchain tasks in a faster and more efficient way.
+
+## CKB-CLI
+
+[ckb-cli](https://github.com/nervosnetwork/ckb-cli) is a CKB command line tool developed in Rust Lang.
+
+If you need to interact with Mainnet blockchain from commandline, `ckb-cli` is a ideal choice to manager accounts, deploy Scripts and send Transactions.
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:18]:
```suggestion
Command-line tools are often used by developers to perform specific blockchain tasks in a faster and more efficient way.

## CKB-CLI

[ckb-cli](https://github.com/nervosnetwork/ckb-cli) is a CKB command-line tool developed in Rust.

If you need to interact with Mainnet blockchain from the command line, `ckb-cli` is an ideal choice to manager accounts, deploy Scripts and send Transactions.
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+    account     Manage accounts
+    mock-tx     Handle mock transactions (verify/send)
+    tx          Handle common sighash/multisig transaction
+    util        Utilities
+    molecule    Molecule encode/decode utilities
+    wallet      Transfer / query balance (with local index) / key utils
+    dao         Deposit / prepare / withdraw / query NervosDAO balance (with local index) / key utils
+```
+
+All second level sub-commands are listed in [wiki page](https://github.com/nervosnetwork/ckb-cli/wiki/Sub-Commands).
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:33]:
```suggestion
All second-level sub-commands are listed on the [wiki page](https://github.com/nervosnetwork/ckb-cli/wiki/Sub-Commands).
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+Show available commands
+
+```shell
+# Top level help doc
+ckb-cli --help
+# RPC help doc
+ckb-cli rpc --help
+```
+
+#### Example: Get live cell (json output format)
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:66]:
```suggestion
#### Example: Get Live Cell (JSON Output Format)
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 9]
```diff
@@ -0,0 +1,269 @@
...
+    },
+    "type": null
+  },
+  "status": "live"
+}
+```
+
+## OffCKB
+
+[offckb](https://github.com/RetricSu/offckb) is a CLI tool developed in Nodejs to help you quickly set up a predefined CKB Devnet and create dApp boilerplate.
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:91]:
```suggestion
[offckb](https://github.com/RetricSu/offckb) is a CLI tool developed in Node.js to help you quickly set up a predefined CKB Devnet and create dApp boilerplates.
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+  sync-config                                       sync offckb.config.ts in your workspace
+  deposit [options] [toAddress] [amountInShannon]   Deposit CKB tokens to address, only devnet and testnet
+  transfer [options] [toAddress] [amountInShannon]  Transfer CKB tokens to address, only devnet and testnet
+  balance [options] [toAddress]                     Check account balance, only devnet and testnet
+  deploy [options]                                  Deploy contracts to different networks, only supports devnet and testnet
+  deployed-scripts [options]                        Show deployed contracts info on networks, only supports devnet and testnet
+  help [command]                                    display help for command
+```
+
+#### Create a full-stack Project
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:128]:
```suggestion
#### Create a Full-Stack Project
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 11]
```diff
@@ -0,0 +1,269 @@
...
+
+Create a new project from predefined boilerplates.
+
+```sh
+offckb create <your-project-name, eg:my-first-ckb-project>
+```
+
+The boilerplate can be targeting on different CKB networks. Check [README.md](https://github.com/nervosnetwork/docs.nervos.org/blob/develop/examples/remix-vite-template/readme.md) in the project to get started.
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:136]:
```suggestion
The boilerplate can target on different CKB networks. Check the [README.md](https://github.com/nervosnetwork/docs.nervos.org/blob/develop/examples/remix-vite-template/readme.md) in the project to get started.
```

File: website/docs/sdk-and-devtool/cli-tool.mdx [hunk 12]
```diff
@@ -0,0 +1,269 @@
+---
+id: cli-tools
+title: CLI Tools
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Tooltip from "@components/Tooltip";
+
+# CLI Tools
```
Comment (yfeng2824) [inline website/docs/sdk-and-devtool/cli-tool.mdx:10]: Should we separate CKB-CLI and OffCKB to different pages (Just like what we do for the SDKs)?

File: website/docs/getting-started/blockchain-networks.mdx [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+  - https://testnet.ckb.dev/rpc
+- Guide: [Running CKB Testnet Node](/docs/node/run-testnet-node)
+- Testnet Explorer: [https://pudge.explorer.nervos.org/](https://pudge.explorer.nervos.org/)
+- Address Prefix: `ckt`, eg `ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqdnnw7qkdnnclfkg59uzn8umtfd2kwxceqxwquc4`
+
+Most of the times, you will need some Testnet tokens to deploy Scripts & test your dApp.
+
+You can get free Testnet tokens from the faucet website:
+
+- [https://faucet-api.nervos.org/](https://faucet-api.nervos.org/)
```
Comment (yfeng2824) [inline website/docs/getting-started/blockchain-networks.mdx:46]:
```suggestion
-  https://faucet.nervos.org/
```

File: website/docs/sdk-and-devtool/cli-tool.mdx (hunk 12 above)
Comment (RetricSu) [reply website/docs/sdk-and-devtool/cli-tool.mdx:10]: I'll give it a thought. let's leave them under cli for now.

**Pull request merged**